ragstack-e2e-tests/pyproject.ragstack-ai.toml

```diff
@@ -22,6 +22,7 @@ pillow = "^10.2.0"
 testcontainers = "^3.7.1"
 python-dotenv = "^1.0.1"
 trulens-eval = "^0.21.0"
+llama-parse = { version = "0.1.4" }
```

Comment:
this is a bad problem because current version of ragstack doesn't support llama-parse. This is because llama-parse wants (>=0.9.40,<0.10.0)

I think we should remove the llama-parse in this file and skip the tests when running this suite. You can try/except the import in the test. If the library is not present, just skip the test